app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,19 @@
+class VotesController < ApplicationController
+
+  def create
+    current_user.votes.create(votable_type: vote_params[0], votable_id: vote_params[1])
```

Comment:
Building on ZP's comments, you should take a look at updating your `HackerNews::VoteLookupService` so that it can handle all the vote lookups.